src/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface Networks {
+  [networkName: string]: Categories;
+}
+
+interface Categories {
+  infra: Addresses;
```

Comment:
Maybe it would be better if we split `infra` into `zns` `zauction`. We've already split out staking